doc/api/deprecations.md

```diff
@@ -356,14 +356,16 @@ The [`fs.lchmodSync(path, mode)`][] API is deprecated.
 
 Type: Documentation-only
 
-The [`fs.lchown(path, uid, gid, callback)`][] API is deprecated.
+The [`fs.lchown(path, uid, gid, callback)`][] API was previously deprecated. As
+of REPLACEME, this API is no longer deprecated.
 
 <a id="DEP0038"></a>
 ### DEP0038: fs.lchownSync(path, uid, gid)
 
 Type: Documentation-only
 
-The [`fs.lchownSync(path, uid, gid)`][] API is deprecated.
+The [`fs.lchownSync(path, uid, gid)`][] API was previously deprecated. As of
```

Comment:
Given it was never hard-deprecated or removed - I'm not sure it's not best to remove this entirely.